.github/workflows/check-changeset.yml

```diff
@@ -2,8 +2,8 @@ name: Check Changeset
 
 on:
   pull_request:
-    branches-ignore:
-      - release
+    branches:
```

Comment:
Note: I just cherry-picked in this fix from master to avoid the spurious GitHub Actions "check changset" failure. See https://github.com/firebase/firebase-js-sdk/pull/7110